src/containers/Transfer.tsx

```diff
@@ -48,26 +47,12 @@ export const Transfer = () => {
   };
 
   const handleTransfer = () => {
-    transfer(parsedAmount.toString()).then((hash) => {
-      if (hash) {
-        setNotification({
-          type: 'success',
-          message: `Transfered!`,
-          link: {
-            href: `${chain?.blockExplorers?.default.url}/tx/${hash}`,
-            text: 'See transaction',
-          },
-          timeout: 5000,
-        });
-
-        resetForm();
-      }
-    });
+    transfer(parsedAmount).then(() => resetForm());
   };
 
   const isValidAddress = useMemo(() => isAddress(inputAddress), [inputAddress]);
   const isBalanceEnough = tokenSelectedBalance && parsedAmount <= tokenSelectedBalance;
-  const isTransferDisabled = !amount || parsedAmount == 0n || !inputAddress || !isValidAddress || !isBalanceEnough;
+  const isTransferDisabled = !amount || parsedAmount == '0n' || !inputAddress || !isValidAddress || !isBalanceEnough;
```

Comment:
`parsedAmount.toString()` is equal '0' not '0n'
```suggestion
  const isTransferDisabled = !amount || parsedAmount == '0' || !inputAddress || !isValidAddress || !isBalanceEnough;
```

Reply:
Nice catch!

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/7/commits/c077427f8925e2e3cff94919ec94d7c94e8b074c